packages/package-store/src/storeController/index.ts

```diff
@@ -98,6 +104,69 @@ export default async function (
     }
   }
 
+  async function findPackageUsages (
+    dependencies: WantedDependency[]
+  ): Promise<FindPackageUsagesResponse[]> {
+
+    /**
+     * If `dependency` has no `alias`, then directly check against `pref`.
+     * This is messy due to overloaded use WantedDependency (npm, github, tarball, etc.)
+     * @param dependency the dependency to parse
+     */
+    function parseDependency (dependency: WantedDependency) {
+      let packageName;
```

Comment:
we don't use semicolons. tslint should've failed on this

Reply:
Weird, I don't see any tslint errors!

```
$ pnpm run lint

> @pnpm/package-store@1.2.0 lint /mnt/d/tejun/Github/pnpm/pnpm/packages/package-store
> tslint -c tslint.json src/**/*.ts test/**/*.ts

Warning: The 'await-promise' rule requires type information.
Warning: The 'deprecation' rule requires type information.
Warning: The 'no-floating-promises' rule requires type information.
Warning: The 'no-unnecessary-qualifier' rule requires type information.
Warning: The 'no-unnecessary-type-assertion' rule requires type information.
Warning: The 'strict-type-predicates' rule requires type information.
Warning: The 'no-use-before-declare' rule requires type information.
```

I'll go ahead and remove semicolons from all my changes though.

Reply:
indeed, there was an issue with the tslint rules. I pushed a fix to master